qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1860,6 +1859,17 @@ def __json_encode__(self):
 
         # Handle non-serializable objects
         json_value["_jobs"] = self._safe_serialize_jobs()
+        service_value = getattr(self, "_service")
+        if service_value:
```

Comment:
This should just be `if self._service`

Reply:
changed the code to be as suggested below:
```python
for attr in ["_service", "_backend"]:
	json_value[attr] = None
	value = getattr(self, attr)
	if value is not None:
		LOG.info("%s cannot be JSON serialized", str(type(value)))
```
Resolving this comment.